### PR TITLE
Music and sfx

### PR DESCRIPTION
**Pull request by ncontinanza**
Closes https://github.com/lambdaclass/curse_of_myrra/issues/322

This PR adds the Araban track to the Araban level, the sound effects related to H4ck and Muflus attacks, a mute/unmute button in the level scene and a volume controller in the pause screen.

Things to test (please test it both in Unity's editor and in a mobile device):
- Start a new game in Araban with H4ck or Muflus, and verify that both the background music and the sfx are muted after starting.
- Use the toggle button in the right hand of the screen to unmute the music and sfx. Make sure that you can hear the background music, and the sound effects when attacking*.
- Use the toggle button to mute the music and sfx. 
- Check that you can control the music volume in the pause screen.

(*): A bug related to the player's state in the server will be overlooked in this PR and solved in an upcoming one: when you move your character and perform an attack, sometimes the sound effect will play and sometimes it won't. Also, when your character is not moving and you perform an attack, you won't hear any sound effects.

### CODE REVIEW

**Review by BertovDev — CHANGES_REQUESTED** [review]
Nice work! I notice that Skill2 of muflus ( Rage ) dont have any sound feedback. Also I feel a little out of timming the Skill1 and Skill3 also in muflus.

Everything else works as expected, nice!

File: client/Assets/Scenes/BackendPlayground.unity
Comment (BertovDev) [inline client/Assets/Scenes/BackendPlayground.unity:1]: There is a Interpolation prefab in this scene out of the canvas. ( Maybe its a merge problem ). Also what else do you change in the scene?

File: client/Assets/Scripts/Skills/Skill.cs [hunk 1]
```diff
@@ -79,22 +89,30 @@ public void ExecuteFeedback()
...
 
     private void SendActionToBackend(RelativePosition relativePosition)
     {
-        ClientAction action = new ClientAction { Action = serverSkill, Position = relativePosition };
+        ClientAction action = new ClientAction
+        {
+            Action = serverSkill,
+            Position = relativePosition
+        };
+        Debug.Log("Sending action to backend");
```
Comment (BertovDev) [inline client/Assets/Scripts/Skills/Skill.cs:104]: Please remove the logs after merge.

File: client/ProjectSettings/ProjectSettings.asset
Comment (BertovDev) [inline client/ProjectSettings/ProjectSettings.asset:1]: Delete this if you can please.

File: client/Assets/Scripts/UI/ToggleAudio.cs
Comment (BertovDev) [inline client/Assets/Scripts/UI/ToggleAudio.cs:1]: This code looks so clean , nice!!

**Review by BertovDev — CHANGES_REQUESTED** [review]

File: client/Assets/Scenes/BackendPlayground.unity
Comment (ncontinanza) [reply client/Assets/Scenes/BackendPlayground.unity:1]: I added the background music (an MMF_Player object) and the mute/unmute button.

**Comment by ncontinanza** [comment]
> Nice work! I notice that Skill2 of muflus ( Rage ) dont have any sound feedback. Also I feel a little out of timming the Skill1 and Skill3 also in muflus.

Talked to @saugon about this, we'll leave it like this by now.

**Review by BertovDev — APPROVED** [review]
Awesome!
I would turn down the music volume a little bit , something like 50% - 70% as default.

- [x] Tested in Unity Desktop
- [x] Tested in Android

**Review by tvillegas98 — APPROVED** [review]
LGTM!

**Pull request merged**